src/interfaces.ts

```diff
@@ -0,0 +1,59 @@
+import { EventObject } from 'dojo-core/interfaces';
```

Comment:
Should we rename `interfaces.ts` to `interfaces.d.ts`. I only see declarations here.